### PR TITLE
Update wording

### PR DESCRIPTION
**Pull request by agsdot**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -14,7 +14,7 @@
   - [[#acknowledgements][Acknowledgements]]
 
 ** Introduction
-   Gkroam is a lightweight [[https://roamresearch.com][Roam Research]] replica, built on top of emacs org-mode. It uses 'ripgrep' to search pages' links and insert references at the bottom of org pages automatically. Almostly, gkroam imitates roam research in as many aspects as possible.
```
Comment (agsdot) [inline README.org:17]: @Kinneyzhang , I have not heard the word "Almostly" used before. The sentence sounded odd with it being used.

File: README.org [hunk 2]
```diff
@@ -55,16 +55,16 @@
...
+   * [[https://github.com/company-mode/company-mode][company]] is used to auto-complete double bracket links and hashtags.
+   * [[https://github.com/BurntSushi/ripgrep][ripgrep(rg)]] is used to search reference links.
 
 ** Tips (important)
 
-   - Install ripgrep(rg) command line firstly and make sure it is in exec-path list.
-   - Do not set =company-idle-delay= variable to zero, it will causes an error. see [[https://github.com/company-mode/company-mode/issues/455][issue#455]].
+   - Install the command line search tool ripgrep(rg) first and make sure it is in the exec-path list.
+   - Do not set the =company-idle-delay= variable to zero, it will causes an error. See [[https://github.com/company-mode/company-mode/issues/455][issue#455]].
    - Use =gkroam-build-caches= to generate all healines' id and cache them in db manually.
```
Comment (agsdot) [inline README.org:68]: I wasn't exactly sure what "healines" was. Was it headings? or Headlines?

- Use gkroam-build-caches to generate all headline ids and cache them in the db manually?
- Use gkroam-build-caches to generate all heading ids and cache them in the db manually?

File: README.org [hunk 3]
```diff
@@ -78,7 +78,7 @@
 *** Main functions
```
Comment (agsdot) [inline README.org:78]: LOC 75 and 76, I'm not sure why it is spelled "aliase" with an "e" at the end. The correct spelling is: https://www.merriam-webster.com/dictionary/alias

**Comment by Kinneyzhang** [comment]
Thanks for your reviewing!

**Comment by agsdot** [comment]
For sure. No problem. I also left two more comments for things I didn't edit. I wasn't exactly sure what you were trying to say for "healines" and for "aliase", I've just never seen that spelling before, rather just "alias". I figure you can just go in later and edit those yourself.

Thanks for creating this Roam Research alternative. It's great having options in the Emacs ecosystem.

**Comment by Kinneyzhang** [comment]
I have worked around the two. They are spelling mistakes.

You're welcome!

**Comment by agsdot** [comment]
@Kinneyzhang  :D

**Pull request merged**